src/main/scala/launcher.scala

```diff
@@ -54,3 +55,25 @@ case object launcher extends LazyLogging {
     }
   }
 }
+
+// The TerminationDaemonBundle in loquat defines its managerCreationTime value
+// querying the getCreatedTime method on the Option returned by
+// aws.as.getGroup(<group-name>). This works only when there is an actual
+// autoscaling group created. As webmiodx uses a local manager that is not
+// running in a external autoscaling group but in the web server itself (see
+// manager.localInstructions(user).run(localTmpDir) above), then the
+// aws.as.getGroup method returns None and the managerCreationTime value of
+// TerminationDaemonBundle in loquat is always None.
+// In order to terminate the instance when a global timeout is reached, we need
+// that the managerCreationTime value holds the actual creation time of the
+// manager. We do that here by overriding the value to an explicit instance of
+// Option[FiniteDuration], defined as the current time in millis.
+class TerminationDaemonBundleWithExplicitCreationTime(
+  override val config: AnyLoquatConfig,
+  override val scheduler: Scheduler,
+  override val initialCount: Int
+) extends TerminationDaemonBundle(config, scheduler, initialCount) {
+
+  override lazy val managerCreationTime = Some(System.currentTimeMillis.millis)
```

Comment:
It's OK in this particular case, but be careful using it anywhere else where the time of instantiating this class and actually launching the pipeline could be different. Probably make this class package-private? or add creation time as a parameter? Not critical, just a suggestion.